### PR TITLE
Document setting gateway poll interval

### PR DESCRIPTION
**Pull request by StephenBarlow**
Followup from #1336

### CODE REVIEW

File: docs/source/managed-federation/uplink.mdx [hunk 1]
```diff
@@ -56,6 +61,21 @@ By default, the gateway retries a single poll request a number of times equal to
...
+
+You can configure the interval at which your gateway polls Apollo Uplink like so:
+
+```js
+const { ApolloGateway } = require('@apollo/gateway');
+
+// ...
+
+const gateway = new ApolloGateway({
+  experimental_pollInterval: 15000 // 15 seconds
```
Comment (trevor-scheer) [inline docs/source/managed-federation/uplink.mdx:74]: I have a PR open (should be landing soon) which deprecates this in favor of `pollIntervalInMs`. Can this PR wait for that change to land? :)

**Review by trevor-scheer — APPROVED** [review]

File: docs/source/managed-federation/uplink.mdx (hunk 1 above)
Comment (trevor-scheer) [reply docs/source/managed-federation/uplink.mdx:74]: Also you _can_ just target my branch and merge this into my PR if you'd like! It'll conveniently get ported to 2.x as well.

`trevor/improve-supergraphSdl`

Comment (StephenBarlow) [reply docs/source/managed-federation/uplink.mdx:74]: Ah yeah sure! Will edit and retarget.

**Comment by StephenBarlow** [comment]
Closing in favor of #1366

**Pull request closed without merging**